pkg/emptydisk/emptydisk.go

```diff
@@ -10,22 +10,25 @@ import (
 	ephemeraldiskutils "kubevirt.io/kubevirt/pkg/ephemeral-disk-utils"
 )
 
-var EmptyDiskBaseDir = "/var/run/libvirt/empty-disks/"
+const emptyDiskBaseDir = "/var/run/libvirt/empty-disks/"
 
```

Comment:
Do we want to take advantage of this PR and move the directory constants out of the individual go files into some centralize place ?  This might help us switching between root/nonroot ..
We can also do it on separate PR ..

Reply:
I agree with you that this should be done, but let's give it a dedicated PR. I would assume that I would now move it to A and a follow up PR just for this would again move it somewhere else.

Reply:
@EdDev maybe "usually", but not in the case we are talking about. I was specifically talking about paths for volumes and disks. 
We have many different volume types,  sometimes implemented in different packages,  today the path is hardcoded in those files/packages. Just changing those to Const and leaving them local in those files won't help much. 
The base directory should be a global value (either through global variable or better a global function exposing the value) , that all other disk/volume related packages should use.  
The full path should be of type "baseDir" + "specificVolumePath"  where the first part is global and the second can be define on the particular package.